src/App.js

```diff
@@ -1,26 +1,46 @@
 import {nanoid} from 'nanoid';
-import {useState, useRef} from 'react';
+import {useState, useEffect, useRef} from 'react';
 import styled from 'styled-components';
 
 import {mazeArray} from './arrays/mazeArray.js';
-import MagicForm from './components/MagicForm/MagicForm';
+import MagicForm from './components/MagicForm/MagicForm.js';
+import MagicImage from './components/MagicImage/MagicImage.js';
 import MagicList from './components/MagicList/MagicList.js';
-import {CodeSpan} from './components/Stylesheet/StyledSpans.js';
 import executeSpell from './functionsfolder/executeSpell.js';
 import findCommandMessage from './functionsfolder/findCommandMessage.js';
+import mazeEnd from './images/mazeRooms/mazeEnd.png';
+import mazeWay from './images/mazeRooms/mazeWay.png';
+import {BackgroundImg} from './stylesheet/StyledImages.js';
+import {CodeSpan} from './stylesheet/StyledSpans.js';
 
 export default function App() {
   const [currentArrayPosition, setCurrentArrayPosition] = useState(mazeArray[0]); //this holds a {object}
 
   function changePosition(newPositionAsString) {
     const newPositionAsObject = mazeArray.find(mazeRoom => mazeRoom.path === newPositionAsString);
     setCurrentArrayPosition(newPositionAsObject);
+    setIsRoomEnlighten(false);
   }
 
   const [savedOrders, setSavedOrders] = useState([]);
 
   const ref = useRef();
 
+  const [isRoomEnlighten, setIsRoomEnlighten] = useState(false);
+  function changeEnlighten() {
+    setIsRoomEnlighten(true);
```

Comment:
Does this function _always_ convert from `true` to `false`? A better and more common/convenient way to write toggle functions such as these would be:
```suggestion
    setIsRoomEnlighten(!isRoomEnlighten);
```

Reply:
That is okay :)